src/test/store/receive-profile.test.js

```diff
@@ -604,24 +604,24 @@ describe('actions/receive-profile', function () {
         store.dispatch(viewProfile(profile));
         expect(getHumanReadableTracks(store.getState())).toEqual([
           'show [process]',
-          '  - show [thread Thread with 190 CPU] SELECTED',
-          '  - hide [thread Renderer]', // <-- hidden
           '  - show [thread DOM Worker]',
-          '  - show [thread Thread with 270 CPU]',
-          '  - show [thread Thread with 330 CPU]',
-          '  - show [thread Thread with 180 CPU]',
-          '  - show [thread Thread with 310 CPU]',
+          '  - hide [thread Renderer]', // <-- hidden
           '  - hide [thread Thread with 0 CPU]', // <-- hidden
-          '  - show [thread Thread with 220 CPU]',
-          '  - hide [thread Thread with 30 CPU]', // <-- hidden
+          '  - show [thread Thread with 110 CPU]',
           '  - show [thread Thread with 120 CPU]',
-          '  - show [thread Thread with 350 CPU]',
           '  - show [thread Thread with 130 CPU]',
-          '  - show [thread Thread with 110 CPU]',
-          '  - show [thread Thread with 230 CPU]',
-          '  - show [thread Thread with 380 CPU]',
           '  - show [thread Thread with 140 CPU]',
+          '  - show [thread Thread with 180 CPU]',
+          '  - show [thread Thread with 190 CPU] SELECTED',
+          '  - show [thread Thread with 220 CPU]',
+          '  - show [thread Thread with 230 CPU]',
+          '  - show [thread Thread with 270 CPU]',
+          '  - hide [thread Thread with 30 CPU]', // <-- hidden
```

Comment:
probably with the change to the natural sort, this test will need to be updated too